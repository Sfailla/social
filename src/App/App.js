import React, { Fragment } from 'react';
import Header from '../Header/Index';

const App = () => {
	return (
		<Fragment>
			<Header />
			{/* 
				<Body />
				<Footer /> 
			*/}
		</Fragment>
	);
};

export default App;
