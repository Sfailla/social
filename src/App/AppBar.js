import React from 'react';
import styled from 'styled-components';
import { FlexCenter } from '../Shared/Styles';

const AppBarStyle = styled.div`
	width: 100%;
	height: 6rem;
	color: black;
	background-color: #00ffd0;
	${FlexCenter};
`;

const Title = styled.h2`
	color: black;
	letter-spacing: 2px;
	text-transform: uppercase;
	font-size: 2rem;
	font-family: Do Hyeon;
`;

const AppBar = () => {
	return (
		<AppBarStyle>
			<Title>Select an option below to get started!</Title>
		</AppBarStyle>
	);
};

export default AppBar;
