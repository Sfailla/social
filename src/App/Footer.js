import React from 'react';
import styled from 'styled-components';
import { MainBG } from '../Shared/Styles';

const FooterStyles = styled.div`
	width: 100%;
	height: 8rem;
	${MainBG};
`;

const Footer = () => {
	return (
		<FooterStyles>
			<h3>This is the footer</h3>
		</FooterStyles>
	);
};

export default Footer;
