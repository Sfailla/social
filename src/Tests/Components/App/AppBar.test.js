import React from 'react';
import '../../setupTests';

import { AppBar } from '../../../App/AppBar';
import { shallow } from 'enzyme';

test('should render <AppBar /> component', () => {
	const wrapper = shallow(<AppBar />);
	expect(wrapper.debug()).toMatchSnapshot();
});
