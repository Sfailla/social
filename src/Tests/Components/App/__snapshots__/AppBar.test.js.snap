// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render <AppBar /> component 1`] = `
"<styled.div>
  <styled.h2>
    Select an option below to get started!
  </styled.h2>
</styled.div>"
`;
